build.sbt

```diff
@@ -39,7 +39,7 @@ lazy val root = Project(appName, file("."))
   .settings(scalaSettings *)
   .settings(defaultSettings() *)
   .settings(
-    scalaVersion := "2.13.12",
+    scalaVersion := "2.13.16",
```

Comment:
Since Scala version is used in 2 places, could you extract it to a constant and re-use it?